src/common/badblock_linux.c

```diff
@@ -155,7 +159,8 @@ os_badblocks_get(const char *file, struct badblocks *bbs)
 			/* check if length is block-aligned */
 			bb_len = ALIGN_UP(bb_len, exts->blksize);
 
-			LOG(4, "bad block found: offset: %llu, length: %llu",
+			LOG(4,
```

Comment:
why level 4 not 10?

Reply:
1) It is more important information
2) There had been level 4 - I did not change it.